packages/viewer/src/components/scene-tree/scene-tree.tsx

```diff
@@ -50,6 +50,9 @@ export class SceneTree {
   @Prop({ reflect: true, mutable: true })
   public controller: SceneTreeController | undefined;
 
+  @Prop({ reflect: true, mutable: true })
+  public client!: SceneTreeAPIClient;
```

Comment:
Do we want this API client to be `reflect`ed here? Not sure it makes sense as a DOM attribute, unless I'm missing something here?

Reply:
I have a follow up PR to address this.